internal/stack/_static/logstash_startup.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -euo pipefail
+
+LOGSTASH_HOME="/usr/share/logstash/"
+
+# logstash expects the key in pkcs8 format.
+# Hence converting the key.pem to pkcs8 format using openssl.
+create_cert() {
+  ls_cert_path="$LOGSTASH_HOME/config/certs"
+  openssl pkcs8 -inform PEM -in "$ls_cert_path/key.pem" -topk8 -nocrypt -outform PEM -out "/tmp/logstash.pkcs8.key"
+  chmod 777 "/tmp/logstash.pkcs8.key"
```

Comment:
Nit. It doesn't use to be a good practice to write a key in `/tmp` or with `777`. But this is a testing scenario, so as you prefer if we don't have another way to do it now.

Reply:
Sounds good! Let me follow up Nits with separate PR because this is somehow blocking my another task: https://github.com/elastic/logstash-filter-elastic_integration/pull/122#discussion_r1487526422